src/start_server.ts

```diff
@@ -81,14 +84,20 @@ export interface ServerOptions {
   additionalRoutes?: Map<string, express.RequestHandler>;
 }
 
-function applyDefaultOptions(options: ServerOptions) {
+function applyDefaultServerOptions(options: ServerOptions) {
   const withDefaults: ServerOptions = Object.assign({}, options);
   Object.assign(withDefaults, {
     port: options.port || 0,
     hostname: options.hostname || 'localhost',
     root: path.resolve(options.root || '.'),
     certPath: options.certPath || 'cert.pem',
     keyPath: options.keyPath || 'key.pem',
+    // TODO(usergenic): The current behavior of polyserve is to use
+    // bower_components for directory and components for url. We should
+    // honor the value of `directory` of `.bowerrc` if found in the root dir
```

Comment:
I would make an exception for `directory` values that reach out of the project directory. That should require a flag given how dangerous it is.